### PR TITLE
Feature/add default templates

### PR DESCRIPTION
**Pull request by devshorts**
Also addresses #1

### CODE REVIEW

File: core/src/main/resources/assets/francois/templates.ts [hunk 1]
```diff
@@ -77,7 +78,35 @@ declare module Foundation {
...
+
+    }
+
+    createTemplate() {
+        this.isCreateable = false;
+        this.francoisApi.createTemplate(this.templateName)
+            .subscribe(r => {
+                console.log(r);
+                this.createSucceeded = true;
+            }, oops => {
```
Comment (devshorts) [inline core/src/main/resources/assets/francois/templates.ts:101]: i like the `oops`

Comment (jakeswenson) [inline core/src/main/resources/assets/francois/templates.ts:101]: :+1: :tongue:

**Comment by jakeswenson** [comment]
#2 fixed as well!! woot

**Comment by jakeswenson** [comment]
:heavy_check_mark: APPROVED!

**Pull request merged**